pkg/okteto/client.go

```diff
@@ -343,6 +343,8 @@ func translateAPIErr(err error) error {
 		return fmt.Errorf("server temporarily unavailable, please try again")
 	case "non-200 OK status code: 401 Unauthorized body: \"\"":
 		return fmt.Errorf("unauthorized. Please run 'okteto context url' and try again")
+	case "non-200 OK status code: 401 Unauthorized body: \"not-authorized: token is expired\\n\"":
```

Comment:
Is it ok to have /n at the end of the error? checking the other graphql errors it doesn't follow the same as the other. Is it on purpose?

Reply:
when the answer comes with a body, the content has a body. this is how the error comes in the cli